docs/templates/model-serving/project.md

```diff
@@ -0,0 +1,90 @@
+# Project/Connection
+
+In Hopsworks [a Project is a sandboxed set](https://www.logicalclocks.com/blog/how-we-secure-your-data-with-hopsworks) of users, data, and programs (where data can be shared in a controlled manner between projects).
+
+Each Project can have its own Model Registry and Model Serving. However, it is possible to [share Model Registry](#sharing-a-model-registry) and Model Serving among projects.
+
+When working with the Model Registry & Serving from a programming [environment](../setup.md) you can connect to a single Hopsworks instance at a time, but it is possible to access multiple Model Registries and Model Servings simultaneously.
+
+A connection to a Hopsworks instance is represented by a [`Connection` object](#connection). Its main purpose is to retrieve the API Key if you are connecting from an external environment.
+
+The [handle](#get_model_registry) can then be used to retrieve a reference to the [Model Registry](../generated/model_registry.md) and the [Model Serving](../generated/model_serving.md) you want to operate on.
+
+## Examples
+
+=== "Python"
+
+    !!! example "Connecting from Hopsworks"
+        ```python
+        import hsml
+        conn = hsml.connection()
+        ```
+
+    !!! example "Connecting from Python environment"
+        To connect from an external Python environment you can provide the api_key_value directly:
```

Comment:
you can -> you have to
api key, project name, and hostname for your Hopsworks cluster

Reply:
Here, we pass the api key with the insecure api_key_value parameter.